decentraland/helpers/storage.js

```diff
@@ -7,6 +7,19 @@ const EPHEMERAL_ACCOUNT_PRIV_KEY = "EPHEMERAL_ACCOUNT_PRIV_KEY";
 const MY_ASSETS_LIST = "MY_ASSETS_LIST";
 const EPHEMERAL_ACCOUNT_CREATION_STATUS = "EPHEMERAL_ACCOUNT_CREATION_STATUS";
 const EPHEMERAL_ACCOUNT_CREATION_ACTIONS = "EPHEMERAL_ACCOUNT_CREATION_ACTIONS";
+const APP_IS_FIRST_USE = "APP_IS_FIRST_USE";
+
+export const storeIsFirstUse = async isFirstUse => {
+  const strIsFirstUse = _toString(isFirstUse);
+  await _storeData(APP_IS_FIRST_USE, strIsFirstUse, false);
```

Comment:
Maybe `IS_FIRST_APP_USE` would make more sense?

Reply:
Resolving this

Reply:
Or wait, I should unresolve this

Reply:
Haha sorry, I was looking at the old commits instead of the new ones. I can resolve this after all.